server/utils/sendNotification.js

```diff
@@ -0,0 +1,34 @@
+const { NEW_NOTIFICATION } = require("../constants");
+const { Notification } = require("../models/NotificationModel");
+/**
+ *
+ * @param {String} sender
+ * @param {String} receiver
+ * @param {Object} payload
+ * @param {string} type
+ * @param {any} context
+ */
+async function sendNotification({ sender, receiver, payload, type, context }) {
```

Comment:
Utility function to reduce boilerplate code

Reply:
I don't like this approach anyways so i think i have to refactor this notification sending system